main/libmicrohttpd/APKBUILD

```diff
@@ -2,15 +2,15 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libmicrohttpd
-pkgver=0.9.62
+pkgver=0.9.63
 pkgrel=0
 pkgdesc="A small C library that is supposed to make it easy to run an HTTP server as part of another application."
 url="https://www.gnu.org/software/libmicrohttpd/"
 arch="all"
 license="GPL"
```

Comment:
License should be `license="LGPL-2.1-or-later"`

Reply:
Done!